src/documentation/pipelines/design-your-pipeline.md

```diff
@@ -31,7 +31,7 @@ The simplest pipelines represent a linear flow of operations, as shown in Figure
 </figure>
 Figure 1: A linear pipeline.
 
-However, your pipeline can be significantly more complex. A pipeline represents a [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) of steps. It can have multiple input sources, multiple output sinks, and its operations (transforms) can output multiple `PCollection`s. The following examples show some of the different shapes your pipeline can take.
+However, your pipeline can be significantly more complex. A pipeline represents a [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) of steps. It can have multiple input sources, multiple output sinks, and its operations (transforms) can read multiple `PCollection`s, then output multiple `PCollection`s. The following examples show some of the different shapes your pipeline can take.
```

Comment:
not a blocker but I think the ordering leaves it a bit unclear if it's the pipeline or the transforms that are outputting multiple PCollections. what about something like this? 
  It can have multiple input sources, multiple output sinks, and its operations (transforms) can both read and output multiple `PCollection`s.